### DISCUSSION TITLE
Eager loading of reverse relationships in templates

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9496] [Ideas]
> Created by: Josh (josh@transom.design) on 2016/04/04 18:17:27 +0000
> Votes at time of UserVoice import: 1

-----------------------------------

**Comment by yoannisj** [comment]
This would be interesting for example to show reverse relations as table attributes in element indexes. It could be used in a `Element::EVENT_PREP_QUERY_FOR_TABLE_ATTRIBUTE` event handler which would be faster than to query for related elements in the `Element::EVENT_SET_TABLE_ATTRIBUTE_HTML` event handler.
